### ISSUE TITLE
Keine Navigation beim gleichen Datum/Uhrzeit

### ISSUE THREAD

**Issue by exscorp** [inspiredminds/contao-sibling-navigation#16]
Hallo, wenn diverse Newsartikel das gleiche Datum / Uhrzeit haben, dann wird bei diesen Artikel keine Navigation angezeigt.

In meinem Fall hatte ich gerade mehrere Newsbeiträge kopiert und in ein anderes Newsarchiv eingefügt. Die neuen (kopierten) Einträge haben alle das gleiche Datum / Uhrzeit. Bei diesem Newsarchiv wird die Navigation nicht angezeigt.

Workaround: Habe bei jedem Eintrag die Uhrzeit geändert, überall +1 Minute. Dann wurde die Navigation wieder gerendert.

Ist das ein Problem der Extension (funktionsweise) oder generelles Contao Problem? Den das gleiche "Problem" tritt auch bei [https://github.com/heimrichhannot/contao-newsnavigation-bundle](url)

**Comment by fritzmg** [comment]
Wenn alle Nachrichten exakt das selbe Datum haben, dann ist es undefiniert welche Nachricht die vorherige oder nächste ist. Man könnte aber willkürlich die ID miteinbeziehen.

**Comment by exscorp** [comment]
Ja, die jeweilige ID miteinbeziehen oder als Fallback wäre gut.

**Comment by jankout** [comment]
ja, das wäre eine gute Lösung. Ansonsten könnte man auch die Uhrzeit checken.

**Comment by fritzmg** [comment]
> Ansonsten könnte man auch die Uhrzeit checken.

If the time is different, then this issue does not occur anyway.
